cargo/cargo_build_script_runner/bin.rs

```diff
@@ -53,6 +53,46 @@ fn run_buildrs() -> Result<(), String> {
     create_dir_all(&out_dir_abs)
         .unwrap_or_else(|_| panic!("Failed to make output directory: {:?}", out_dir_abs));
 
+    // Symlink the execroot to the manifest_dir so that we can use relative paths in the arguments.
+    let exec_root_paths = std::fs::read_dir(&exec_root)
+        .map_err(|err| format!("Failed while listing exec root: {:?}", err))?;
+    for path in exec_root_paths {
+        let path = path
+            .map_err(|err| {
+                format!(
+                    "Failed while getting path from exec root listing: {:?}",
+                    err
+                )
+            })?
+            .path();
+        let metadata = std::fs::metadata(&path).map_err(|err| {
+            format!(
+                "Failed while getting metadata for path {:?}: {:?}",
+                path, err
+            )
+        })?;
+
+        // Only symlink directories.
```

Comment:
Why not symlink files too?

Reply:
There's actually no reason not to symlink in files as well. I did not need it for my use case so I ignored it. I'll get file symlinks added in.